server/routes/github/github_repo_selection.py

```
@@ -88,16 +79,19 @@ def save_repo_selections(user_id):
         if not isinstance(repositories, list):
             return jsonify({"error": "repositories array is required"}), 400
 
-        org_id = _get_user_org_id(user_id)
+        org_id = _resolve_org(user_id)
+        predicate, pred_params = _org_read_predicate(user_id, org_id)
 
         with db_pool.get_admin_connection() as conn:
             with conn.cursor() as cur:
```

Ruff (0.15.12)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)